python_modules/libraries/dagster-databricks/dagster_databricks/configs.py

```diff
@@ -589,16 +589,14 @@ def _define_adls2_storage_credentials():
     )
 
 
-def _define_storage_credentials():
-    return Selector(
-        {"s3": _define_s3_storage_credentials(), "adls2": _define_adls2_storage_credentials()},
-    )
-
-
 def define_databricks_storage_config():
     return Field(
         Selector(
-            {"s3": _define_s3_storage_credentials(), "adls2": _define_adls2_storage_credentials()}
+            {
+                "s3": _define_s3_storage_credentials(),
+                "adls2": _define_adls2_storage_credentials(),
+                "N/A": Field(Permissive(), description="Use if no storage configuration necessary"),
```

Comment:
would you ever actually configure something within this "N/A" field? What's the difference between specifying the N/A field and just not configuring storage at all? If there's no difference, I'd recommend just removing the N/A field

Reply:
Gave it a shot locally but unfortunately doesn't do the trick (same `TypeError: __init__() missing 1 required positional argument: 'storage'` error).. Any other suggestions come to mind?
```
def __init__(
        self,
        run_config,
        permissions,
        databricks_host,
        databricks_token,
        secrets_to_env_variables,
        staging_prefix,
        wait_for_logs,
        max_completion_wait_time_seconds,
        env_variables={},
        storage={},
        poll_interval_sec=5,
        local_pipeline_package_path=None,
        local_dagster_job_package_path=None,
    ):
```

Reply:
hi @Phazure I can't see why you'd be hitting that error -- do you mind showing a bit more of the stack trace? Is this coming from the constructor of the DatabricksPysparkStepLauncher class or somewhere else? I'm wondering if that's an issue with the code being out of sync with wherever you're running it from (i.e. the file hasn't saved or something), just because in the code you have, `storage` is definitely not a required positional argument to that class.

Reply:
@OwenKephart Ah yes I rescind my statement - this does indeed work. Forgot to include `--force-reinstall` in the `pip install` command after making the new changes. Sorry for the confusion!